.github/workflows/ost.yml

```diff
@@ -0,0 +1,17 @@
+name: OST
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  trigger-ost:
+    if: |
+      ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/ost') &&
+          ( github.event.comment.author_association == 'OWNER' ||
```

Comment:
I remember some discussion about the OWNER is this role really needed? OWNER should be oVirt org which I guess won't push PR. But don't want to block it so we can adjust in follow up.

Reply:
We need a link to the docs explaining what is OWNER, MEMBER, and COLLABORATOR

Reply:
https://docs.github.com/en/graphql/reference/enums#commentauthorassociation

Reply:
So from the description it seems we only need the MEMBER and COLLABORATOR.

Reply:
fixed